### DISCUSSION TITLE
SO 2.3.130 inaccessible... Browser redirecting to just hostname and not FQDN

### DISCUSSION THREAD

**Discussion by steelwing** [Security-Onion-Solutions/securityonion#8173] [Unsupported Versions] [answered]
So I updated to 2.3.130, and now when I go by browser to my SO host, it first prompts me that it's not a secure link and then I have to click "Proceed" to go ahead.  But the redirect after that doesn't use the host's FQDN, it uses the hostname itself, and in my environment the hostname doesn't resolve to the right IP.  Even if I type in the IP directly, after clicking proceed, it takes me to the hostname.  What can I change to remove this behavior?

**Comment by Sahale** [comment, marked as answer]
Hey!
Looks like you need to follow this [doc](https://docs.securityonion.net/en/2.3/url-base.html). Another _workaround_ option is to edit hosts file on your system.

**Reply by steelwing** [reply]
:D Thanks!  That did it!
